output/webrtc/webrtc.cc

```diff
@@ -471,6 +490,38 @@ static void http_webrtc_remote_candidate(http_worker_t *worker, FILE *stream, co
   http_write_response(stream, "200 OK", "application/json", "{}", 0);
 }
 
+static void *heartbeat_checker(void *)
+{
+  uint64_t disconnect = 10;
+  while (true) {
+    {
+      uint64_t now_s = get_monotonic_time_us(NULL, NULL) / 1000 / 1000;
+      std::unique_lock lk(webrtc_clients_lock);
+      auto it = webrtc_clients.begin();
+      while (it != webrtc_clients.end()) {
+        auto client = *it;
+        if (!client) {
+          continue;
+        }
+
+        uint64_t heartbeat_detla = now_s - client->last_heartbeat_s;
+        if (heartbeat_detla >= disconnect) {
+          LOG_INFO(client.get(), "No heartbeat from client, removing.");
+          it = webrtc_clients.erase(it);
+        } else {
+          if (heartbeat_detla > disconnect / 2) {
+            LOG_DEBUG(client.get(), "Checking if client still alive.");
+            client->dc->send("ping");
```

Comment:
When I refresh the /webrtc page too quickly I've noticed this line throws an exception crashing the camera-streamer:

```
terminate called after throwing an instance of 'std::runtime_error'
  what():  DataChannel is closed
Aborted
```